src/nodes.coffee

```diff
@@ -4822,51 +4873,107 @@ isLocationDataEndGreater = (a, b) ->
 # encompasses the location data of both nodes. So the new `first_line` value
 # will be the earlier of the two nodes’ `first_line` values, the new
 # `last_column` the later of the two nodes’ `last_column` values, etc.
-mergeLocationData = (locationDataA, locationDataB) ->
+# 
+# If you only want to extend the first node’s location data with the start or
```

Comment:
Now `mergeLocationData()` (as opposed to `mergeAstLocationData()`) needs `justLeading`/`justEnding`, so updated its comment and added tests like I'd done for `mergeAstLocationData()`